openc3/lib/openc3/api/limits_api.rb

```diff
@@ -43,41 +42,6 @@ module Api
                        'get_limits_events',
                      ])
 
-    # Get the list of stale packets for a specific target or pass nil to list
-    # all stale packets
-    #
-    # @param with_limits_only [Boolean] Return only the stale packets
-    #   that have limits items and thus affect the overall limits
-    #   state of the system
-    # @param target_name [String] The target to find stale packets for or nil to list
-    #   all stale packets in the system
-    # @param staleness_sec [Integer] The amount of time to pass before a packet is marked stale
-    # @return [Array<Array<String, String>>] Array of arrays listing the target
-    #   name and packet name
-    def get_stale(with_limits_only: false, target_name: nil, staleness_sec: 30, scope: $openc3_scope, token: $openc3_token)
```

Comment:
Just no, or is there an alternative to get the list of stale packets?

Reply:
Just no for now.   We could do a method for checking an individual packet, but these check everything methods do not scale.